packages/rainbowkit/src/index.ts

```diff
@@ -14,8 +14,13 @@ export {
   RainbowKitAuthenticationProvider,
   createAuthenticationAdapter,
 } from './components/RainbowKitProvider/AuthenticationContext';
-export type { Wallet, WalletList } from './wallets/Wallet';
-export type { Chain } from './components/RainbowKitProvider/RainbowKitChainContext';
+export type { RainbowKitChain as Chain } from './components/RainbowKitProvider/RainbowKitChainContext';
+export type {
+  Wallet,
+  WalletList,
+  WalletDetailsParams,
+  RainbowKitWalletConnectParameters,
```

Comment:
Good to expose this into rainbowkit so people can use these types if they want to build their own wallets